src/inversify.config.ts

```diff
@@ -29,8 +29,9 @@ export function initContainer(): Container {
   container.bind<Optional<string>>(TYPES.NODE_ENV).toConstantValue(getEnv('NODE_ENV'))
   container.bind<string>(TYPES.CDN_HOST).toConstantValue(getEnvOrThrow('CDN_HOST'))
   container.bind<string>(TYPES.CHS_COMPANY_PROFILE_API_LOCAL_URL).toConstantValue(getEnvOrThrow('CHS_COMPANY_PROFILE_API_LOCAL_URL'))
-  container.bind<Optional<string>>(TYPES.PIWIK_SITE_ID).toConstantValue(getEnv('PIWIK_SITE_ID'))
   container.bind<Optional<string>>(TYPES.PIWIK_URL).toConstantValue(getEnv('PIWIK_URL'))
+  container.bind<Optional<string>>(TYPES.PIWIK_SITE_ID).toConstantValue(getEnv('PIWIK_SITE_ID'))
+  container.bind<Optional<string>>(TYPES.PIWIK_LANDING_PAGE_START_GOAL_ID).toConstantValue(getEnv('PIWIK_LANDING_PAGE_START_GOAL_ID'))
```

Comment:
Should these be optional?

Reply:
I believe they are optional because in the nunjucks middleware, we check if the piwik env vars are set.
But yeah don't think we need that if check though, piwik should always be set.

Reply:
I think we should expect piwik to be enabled by default. So these should be mandatory